lacework/resource_lacework_policy_compliance.go

```diff
@@ -33,6 +33,13 @@ func resourceLaceworkPolicyCompliance() *schema.Resource {
 				Required:    true,
 				Description: "The id of the query",
 			},
+			"query_language": {
+				Type:        schema.TypeString,
+				Required:    false,
+				Optional:    true,
+				Computed:    true,
```

Comment:
This computed value needs to be set with `d.Set("query_language", response.Data.QueryLanguage)` too I think.